src/freenet/clients/http/Toadlet.java

```diff
@@ -69,8 +69,15 @@
 	 * this function is necessary.
 	 * 
 	 * @param context Can be used to decide the return value, for example to check session cookies using {@link SessionManager}.
-	 * @return The result of {@link #showAsToadlet()}, which is <code>this</code> by default. Override this function to return something else for invisible
-	 *         Toadlets as explained above.
+     * @return
+     *     The result of {@link #showAsToadlet()}, which is <code>this</code> by default.<br>
+     *     The result of that deprecated function is returned instead of directly returning
+     *     <code>this</code> to keep legacy Toadlet implementations working which aim to override
+     *     the showAsToadlet() mechanism but only override the old function instead of this
+     *     one here.<br><br>
```

Comment:
Thanks. Hmm... your suggested rephrase is not 100%-well-defined I think.
It is not clear what the "This" in "This is.." means. The meaning of "the fact that we return the value of a deprecated function instead of returning the value itself directly" should be somehow suggested.

Here's another try:
```
* The result of {@link #showAsToadlet()}. It returns <code>this</code> by default.<br>
* The result of that deprecated function is returned instead of directly returning
* <code>this</code> to keep legacy Toadlets working as they might override the old
* function.<br><br>
```

Would that one be OK to you?